docs/docs/administration-and-security/data-management/import-from-launchdarkly.md

```diff
@@ -65,3 +65,15 @@ Multivariate LaunchDarkly flags will be imported into Flagsmith as MultiVariate
 Multivariate values will be taken from the `variations` field of within LaunchDarkly.
 
 Values set to serve when targeting is off will be imported as control values.
+
+#### Archived and deprecated flags
+
+Archived flags in LaunchDarkly are imported as archived flags in Flagsmith.
+
+Deprecated flags in LaunchDarkly are also imported as archived flags in Flagsmith. They receive a "Deprecated" tag to distinguish them from flags that were archived in LaunchDarkly.
+
+:::warning
+
+In LaunchDarkly, deprecated flags continue to evaluate normally. In Flagsmith, archived flags return fallback values. Unarchive deprecated flags after import if older application versions still request them.
+
+:::
```

Comment:
@matthewelwell FYI, this could invalidate this PR, or at least the approach we took so far. Let's discuss.

Reply:
No it's not. The engine does not have or receive any concept of archived features. I was fooled by a shallow AI assumption, I'm sorry.